modules/core/src/lib/picking/pick-info.js

```diff
@@ -21,17 +21,28 @@
 // TODO - break this monster function into 3+ parts
 /* eslint-disable max-depth, max-statements */
 
-export function processPickInfo({
-  pickInfo,
-  lastPickedInfo,
-  mode,
-  layers,
-  viewports,
-  x,
-  y,
-  z,
-  pixelRatio
-}) {
+export function getEmptyInfo({pickInfo, viewports, pixelRatio, x, y, z}) {
```

Comment:
Looks good, would appreciate a JSDoc header explaining what this does / why we need this. We don't want to return null, because we still want to return coordinates. 

Assuming this was the case before as well otherwise it seems like a breaking change. Some of this may be old but I really don't remember anymore so helpful to spell out the decisions we made.

Reply:
`getEmptyPickInfo`, `getNullPickInfo` etc?

Reply:
> Should it be documented

This is internal. Return values of `deckPicker.pickObject` are handled by `deck.pick*` before the user sees them.

> Assuming this was the case before as well 

Correct, there is no change to the behavior when there are pickable layers. This function simply moves some code out from `processPickingInfo`.